src/NServiceBus.Core/Pipeline/BehaviorContext.cs

```diff
@@ -19,6 +20,11 @@ public IBuilder Builder
             }
         }
 
+        public Task RaiseNotification<T>(T notification) where T : INotification
+        {
+            return Builder.Build<EventAggregator>().Raise(notification);
```

Comment:
This shows that we have no clear concept how to manage dependencies internally. We talked about the service locatorish API internally. A workaround here would be to abuse the extension dictionary until we have it.